doc/getting-started.md

```diff
@@ -0,0 +1,344 @@
+## Getting started
+
+Your [Dogecoin Core installation](/INSTALL.md) is done, you now have `dogecoind` or `dogecoin-qt` executables available to run a node, and `dogecoin-cli`/`dogecoin-tx` tools but don't know how it works ?
+
+This tutorial will help you to go through the basics to use them !
+
+> **Warning :** For simplicity, explained commands assume executables are under `PATH` environment variable.  
+> If needed, specify their location by doing `PATH=$PATH:/path/to/executables` or use their path as command like :
+> ```bash
+> $ /path/to/dogecoin-cli [arguments ...]
+> ```
+
+### Table of contents
+
+1. [Run a dogecoin node](#run-a-dogecoin-node)
+2. [Introduction to JSON-RPC API](#introduction-to-json-rpc-api)
+    * [Create a wallet](#create-a-wallet)
+    * [Send a transaction](#send-a-transaction)
+    * [Inspect blocks and transactions](#inspect-blocks-and-transactions)
+3. [Node configuration](#node-configuration)
+    * [Mainnet, testnet and regnet](#mainnet-testnet-and-regnet)
+    * [RPC credentials](#rpc-credentials)
+    * [Ports](#ports)
+    * [Memory](#memory)
+    * [Mode debug](#mode-debug)
+
+## Run a dogecoin node
+
+To launch your node, you can choose `dogecoind` to run an headless server or `dogecoin-qt` for a server with a GUI.
+
+In a terminal, run :
+```bash
+# Headless node
+$ dogecoind -printtoconsole
+
+# GUI
+$ dogecoin-qt
+```
```

Comment:
It's because commands are listed on the previous sentence, you would remove it ? I think it's great to keep a block of code here to make it more obvious.

Otherwise you're suggesting 2 blocks with a small sentence before each of them ?

Reply:
> Otherwise you're suggesting 2 blocks with a small sentence before each of them ?

Yeah. Now it's a code block with comments - it looks odd and confusing.

Reply:
Updated, tell me if it's ok for you.

Reply:
It's still the same for me?

Reply:
My bad, I had merge conflict.

Reply:
Yes, this is better!